core/src/lib.rs

```diff
@@ -635,7 +635,7 @@ pub type IdentMap = BTreeMap<NamespacedKeyword, Entid>;
 pub type EntidMap = BTreeMap<Entid, NamespacedKeyword>;
 
 /// Map attribute entids to `Attribute` instances.
-pub type SchemaMap = BTreeMap<Entid, Attribute>;
```

Comment:
What is the justification?  Is this to be symmetric with `ident_map`?

Reply:
Mainly because this is a mapping from entid to `Attribute`, so this is an `AttributeMap` — "Schema" is the thing that holds an `EntidMap`, an `IdentMap`, and an `AttributeMap`!

But more broadly, I'm trying to shift wording to a place where most of Mentat knows about the "schema" — a collection of attributes and idents — and most of its consumers care about "vocabulary", which are versioned, documented, named collections of attributes and migrations. The more precisely we can use the word "schema", the clearer this distinction gets; up to now we've been a little murky.